#!/bin/sh
source="$(find /usr/lib/python*.*/site-packages/configgen/generators/citra -name citraGenerator.py)"
cp -a "${source}" "${source}.backup-cemuhook"
patch "${source}" << 'EOF'
--- ${source}
+++ citraGenerator.py
@@ -56,7 +56,6 @@
         citraConfig = configparser.RawConfigParser(strict=False)
         citraConfig.optionxform=str             # Add Case Sensitive comportement
         if os.path.exists(citraConfigFile):
-            os.remove(citraConfigFile)          # Force removing qt-config.ini
             citraConfig.read(citraConfigFile)

         ## [LAYOUT]
EOF

echo >&2 "Citra Generator was patched for cemuhook."
